azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -82,6 +80,15 @@ internal class CallingSDKWrapper(
             }
         }
 
+    private val callClient: CallClient
+        get() {
+            try {
```

Comment:
don't use exception as flow control.
https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why

Can remove in kotlin pretty cleanly I think just.

callClientInternal ?: throw CallCompositeException("Call is not started", IllegalStateException())

Reply:
As per discussion we will fix for GA.